14052015-reverse-polish-nishant

NishantUpadhyay-BTC opened the pull request.

Commit:
Solve problem Reverse-polish notation

Commit:
Use 'empty?' method instead of comapare directly with ''

subhash-shah commented on 14-05-2015/reverse-polish/reverse_polish.txt:
```diff
@@ -0,0 +1,20 @@
+class ReversePolish
+  def self.evaluate(polish_str)
+    @res_arr = []
+    return 0 if polish_str.empty?
```
This should be the first statement in your method.

subhash-shah commented on 14-05-2015/reverse-polish/reverse_polish.txt:
```diff
@@ -0,0 +1,20 @@
+    @res_arr = []
+    return 0 if polish_str.empty?
+    elements= polish_str.split(' ')
+    elements.each do |ele|
```
`elements` is not used except in this line. You can skip to create local variable `elements`.

Commit:
Refactor code as per comments.
- Remove local variable 'elements' which is only used once.
- Condition placed at the top for checking the input is empty or not?.

SubhashShah-BTC commented:
@NishantUpadhyay-BTC : Can you please merge this pull request?
